scripts/process/process.storm_gage_height.R

```diff
@@ -11,17 +11,16 @@ process.storm_gage_height <- function(viz = as.viz("storm-gage-height")) {
   
   gage_height <- deps[["gage-height"]]
   
-  
-  gage_mins <- gage_height %>% 
+  gage_nils <- gage_height %>% 
     filter(p_Inst != no_value) %>%
     group_by(site_no) %>%
-    summarize(min = min(p_Inst))
+    summarize(nil = (min(p_Inst)-0.1)) # this -0.1 makes these mins unique but not mess with scaling.
```

Comment:
This feels like a bit of a hack, but I think it'll work pretty well. Gives us a unique value to mask but doesn't mess with the plotting as it stands.